config.js

```diff
@@ -43,16 +43,10 @@ module.exports = {
   govukNotify: {
     notifyApiKey: process.env.NOTIFY_KEY || '',
     templateUserAuthId: process.env.TEMPLATE_USER_AUTHORISATION_ID || 'fe0408cb-24f0-4b5f-9bdb-6569834039fb',
-    templatePDF: process.env.TEMPLATE_PDF || 'd48b84c8-9350-4cf8-b31c-8cc4ef8237f1',
```

Comment:
Hang on, are we deleting this email entirely, or are we replacing it with an email that just doesn't have a PDF?

Reply:
I'll double check